stan/math/prim/mat/fun/cov_exp_quad.hpp

```diff
@@ -18,6 +19,8 @@ namespace math {
 /**
  * Returns a squared exponential kernel.
  *
+ * @deprecated use <code>gp_exp_quad_cov</code>
```

Comment:
Function with @deprecated usually have other comments removed (no explanation of what the function does, no @tparam, etc). See https://github.com/stan-dev/math/blob/develop/stan/math/prim/mat/prob/dirichlet_log.hpp#L37